db.sql

```diff
@@ -52,7 +52,7 @@ CREATE TABLE `keysets` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `keysetid` varchar(50) NOT NULL,
   `kskeys` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin DEFAULT NULL,
-  `polid` int(11) NOT NULL,
+  `polid` int(11) DEFAULT NULL,
```

Comment:
- Why is this change required?
- What happens to the rows without a `polid`? Won't they just grow forever, consuming more space that isn't strictly necessary?
- Please could you split this PR into two commits, adding some detail on the rationale for the `polid` change.

Reply:
Policies are optional for all entities. There is one default policy for the entire KBS and then any policies that are added for a keyset are added on top of that and any policies that are added for individual keys are added on top of that. This is why `get_secret_policy` and `get_keyset_policy` return Options rather than Results. No design change here we'd just never tested a keyset without a policy. 

I'm not sure about your second point. Keysets should be added or removed by the guest owner. I can put the db changes in a separate commit.